packages/scripts/README.md

```diff
@@ -16,6 +16,8 @@ You only need to install one npm module:
 npm install @wordpress/scripts --save-dev
 ```
 
+**Note**: This package requires `npm` 6.9.0 or later, and is not compatible with older versions.
```

Comment:
Should this also mention Node.js v10 also?

It is noted as required in the changelog here https://github.com/WordPress/gutenberg/pull/18048/files#diff-748355c268adfa7ea2255a3045d81ed9R5

Reply:
Yes, we can extend it.